modules/openapi-generator/src/main/resources/python/configuration.mustache

```diff
@@ -462,12 +463,15 @@ conf = {{{packageName}}}.Configuration(
         auth = {}
 {{#authMethods}}
 {{#isApiKey}}
-        if '{{keyParamName}}' in self.api_key:
+        if '{{name}}' in self.api_key{{#vendorExtensions.x-auth-id-alias}} or '{{.}}' in self.api_key{{/vendorExtensions.x-auth-id-alias}}:
             auth['{{name}}'] = {
                 'type': 'api_key',
                 'in': {{#isKeyInCookie}}'cookie'{{/isKeyInCookie}}{{#isKeyInHeader}}'header'{{/isKeyInHeader}}{{#isKeyInQuery}}'query'{{/isKeyInQuery}},
                 'key': '{{keyParamName}}',
-                'value': self.get_api_key_with_prefix('{{keyParamName}}')
+                'value': self.get_api_key_with_prefix(
+                    '{{name}}',{{#vendorExtensions.x-auth-id-alias}}
+                    alias='{{.}}',{{/vendorExtensions.x-auth-id-alias}}
```

Comment:
@spacether please note that the `alias`argument is only generated for `x-auth-id-alias` now.

Reply:
Why not get rid of the alias parameter and pass in the alias value as the identifier value? When we have the alias value don't we never use the identified value?

Reply:
I am trying to keep the code backward compatible. In general, it's easier to keep backward compatibility with keyword arguments especially when they have default value.

Reply:
I agree that the approach in the PR would be my preference as well.

Reply:
Understood, thanks